delegation/src/Pos/Delegation/Logic/Mempool.hs

```diff
@@ -52,7 +53,9 @@ import           Pos.Util.Concurrent.PriorityLock (Priority (..))
 getDlgMempool
     :: (MonadIO m, MonadDBRead m, MonadDelegation ctx m, MonadMask m)
     => m DlgPayload
-getDlgMempool = UnsafeDlgPayload <$> (runDelegationStateAction $ uses dwProxySKPool HM.elems)
+getDlgMempool =
+    fmap (UnsafeDlgPayload . toList) $
+    runDelegationStateAction $ uses dwProxySKPool (S.fromList . HM.elems)
```

Comment:
Revert these changes please, you convert to a `Set` and then back to a list.

Reply:
Once again, revert it please.

Reply:
Done.